_scripts/instruction-widget/install.js

```diff
@@ -28,7 +28,7 @@ module.exports = function(context) {
 
     // This is the list of distributions that should be shown our snap
     // instructions.
-    snap_distros = ["snap", "ubuntu", "arch", "opensuse", "fedora"];
+    snap_distros = ["snap", "ubuntu", "arch", "opensuse", "fedora", "debian"];
```

Comment:
By the way, I missed that in the previous PR. Could you add the `var` keyword on this variable declaration, to avoid it to be global?

Reply:
I don't love blocking PRs on fixing problems the PR didn't introduce, but this is done.